README.md

```diff
@@ -12,7 +12,7 @@ by gradually shifting traffic to the new version while measuring metrics and run
 ![flagger-overview](https://raw.githubusercontent.com/fluxcd/flagger/main/docs/diagrams/flagger-canary-overview.png)
 
 Flagger implements several deployment strategies (Canary releases, A/B testing, Blue/Green mirroring)
-using a service mesh (App Mesh, Istio, Linkerd) or an ingress controller (Contour, Gloo, NGINX, Skipper, Traefik) for traffic routing.
+using a service mesh (App Mesh, Alibaba Cloud Service Mesh, Istio, Linkerd) or an ingress controller (Contour, Gloo, NGINX, Skipper, Traefik) for traffic routing.
```

Comment:
I’m not ok with this change, ASM is covered by Istio in this list.

Reply:
> I’m not ok with this change, ASM is covered by Istio in this list.

OK, I will follow your suggestion in this pull, and to explain why in new pr.